functions-common

```diff
@@ -1542,7 +1542,7 @@ function _run_under_systemd {
     fi
 
     $SYSTEMCTL enable $systemd_service
-    $SYSTEMCTL start $systemd_service
+    $SYSTEMCTL restart $systemd_service
```

Comment:
Heh, this is funny how many times this has been switched upstream.  I prefer using restart but some hate seeing the log messages when the service is not already running...